test/release/examples/primers/learnChapelInYMinutes.chpl

```diff
@@ -523,10 +563,65 @@ proc whereProc( param N : int ): void
 
 whereProc( 10 );
 whereProc( -1 );
-// whereProc( 0 ) would result in a compiler error because there
-// are no functions that satisfy the where clause's condition.
-// We could have defined a whereProc without a where clause that would then have
-// served as a catch all for all the other cases (of which there is only one).
+
+// ``whereProc( 0 )`` would result in a compiler error because there
+// are no functions that satisfy the ``where`` clause's condition.
+// We could have defined a ``whereProc`` without a ``where`` clause
+// that would then have served as a catch all for all the other cases
+// (of which there is only one).
+
+/*
+.. primers-yminutes-intent:
+
+Intents
+-------
+*/
+
+/* Intent modifiers on the arguments convey how those arguments are passed to the procedure.
```

Comment:
Something about this list causes the first line to be bolded... Maybe new lines  will fix it?